### PR TITLE
Use pagination for the /exchanges query

### PR DESCRIPTION
**Pull request by measure-fi**

### CODE REVIEW

File: gecko.py [hunk 1]
```diff
@@ -44,31 +45,25 @@ def fetch():
...
-    df = exchanges(dex)
-    if n_item > 1:
-        for i in range(n_item):
-            tmp_df = exchanges(dex)
-            for j in tmp_df.index:
-                if j not in df.index:
-                    df.loc[j, "volume"] = tmp_df.loc[j, "volume"]
-
-    return df
-
```
Comment (fieldtheory123) [inline gecko.py:70]: Why remove this function? It is meant for reduce the potential instability of data coming from cg endpoints.

**Comment by fieldtheory123** [comment]
Overall I agree with the idea of doing paginated query instead of single-paged. We probably do not have the demand for multiple pages of pool data but it could be useful in the long term. Only question is regarding removing the multi-query function.

File: gecko.py (hunk 1 above)
Comment (measure-fi) [reply gecko.py:70]: The point of this PR is that I have discovered the cause of the instability, and we can correct it by reading multiple pages.

**Comment by taariq** [comment]
@fieldtheory123 do we need this or we can close?

**Comment by fieldtheory123** [comment]
We don't need it. I'm closing it.

**Pull request closed without merging**